internal/test_cases/run_test_case.go

```diff
@@ -17,8 +18,45 @@ import (
 // It is sent to the "run" command of the executable,
 // the expected outputs are generated using the lox.Run function,
 // With that the output of the executable is matched.
+
```

Comment:
In Go, it's uncommon to have a newline after a comment that describes a type - would stick to what we're doing in other areas of the codebase.